app/views/govuk_publishing_components/components/_select.html.erb

```diff
@@ -3,6 +3,7 @@
   id ||= false
   label ||= false
   full_width ||= false
+  name_attr ||= id
```

Comment:
If the name defaults to the `id`, what happens to the name when the `id` defaults to false?